README.md

```diff
@@ -114,3 +114,92 @@ Specifically, it will emit 2 kinds of span for each view that is wrapped:
 
 `View Body` spans encompass just the `body()` call of `HoneycombInstrumentedView, and include the following attributes:
 - `ViewName` (string): the name passed to `HoneycombInstrumentedView` 
+
+### SwiftUI Navigation Instrumentation
+iOS 16 introduced two [new Navigation types](https://developer.apple.com/documentation/swiftui/migrating-to-new-navigation-types) that replace the now-deprecated [NavigationView](https://developer.apple.com/documentation/swiftui/navigationview).
+
+We offer a convenience view modifier (`.instrumentNavigations(path: String)`) for cases where you are using a [`NavigationStack`](https://developer.apple.com/documentation/swiftui/navigationstack) and [managing navigation state externally](https://developer.apple.com/documentation/swiftui/navigationstack#Manage-navigation-state)
+
+ex.:
+```swift
+import Honeycomb
+
+struct SampleNavigationView: View {
+    @State private var presentedParks: [Park] = [] // or var presentedParks = NavigationPath()
+
+    var body: some View {
+        NavigationStack(path: $presentedParks) {
+            List(PARKS) { park in
+                NavigationLink(park.name, value: park)
+            }
+            .navigationDestination(for: Park.self) { park in
+                ParkDetails(park: park)
+            }
+        }
+        .instrumentNavigations(path: presentedParks) // convenience View Modifier
+    }
+}
+```  
+
+Whenever the `path` variable changes, this View Modifier will emit a span with the name `Navigation`. This span will contain the following attributes:
```

Comment:
We should probably make these names consistent with the UIKit navigation instrumentation, right? I'm worried that all our different trace names look different.

Reply:
ie. emitting `viewDidAppear` as the span name here?

Reply:
I guess? Would they stomp on each other? I guess we'll find out from the smoke tests.